### PR TITLE
View all posts

### PR DESCRIPTION
**Pull request by seaborndan**
Closes issue #45 can view all approved posts published before the current time in descending order, displays author displayname

Tested by adding in post published in the future, and by viewing which posts are approved and unapproved within the database

### CODE REVIEW

File: Tabloid/appsettings.json [hunk 1]
```diff
@@ -9,5 +9,6 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "DefaultConnection": "server=localhost\\SQLExpress;database=Tabloid;integrated security=true"
-  }
+  },
+  "FirebaseProjectId": "tabloid-4bf15"
```
Comment (joshdbarton) [inline Tabloid/appsettings.json:13]: This line should be in appsettings.Local.json, not here.

**Review by joshdbarton — CHANGES_REQUESTED** [review]

File: Tabloid/client/src/index.js [hunk 2]
```diff
@@ -15,7 +14,6 @@ firebase.initializeApp(firebaseConfig);
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <React.StrictMode>
-    <Header />
```
[the remark this answers is not in the document]
Comment (seaborndan) [reply Tabloid/client/src/index.js:18]: This was a duplicate that created merge issues, app.js has the header route in it already

File: Tabloid/appsettings.json (hunk 1 above)
Comment (seaborndan) [reply Tabloid/appsettings.json]: Fixed!

**Review by joshdbarton — APPROVED** [review]
:shipit:

**Pull request merged**